deps/aslam_cameras_april/test/logging/logTest.cpp

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Yup, it is run! There were some other tests that I didn't include from this folder, as the files didn't include the relevant headers.